### PR TITLE
activation user on login

### PR DESCRIPTION
**Pull request by andrei-anfilets**

### CODE REVIEW

File: src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        private readonly IMemberService _memberService;
+
+        public UserLoginAuthorizationHandler(IMemberService componentContext)
+        {
+            _memberService = componentContext;
+        }
+
+        public async Task HandleAsync(AuthorizationHandlerContext context)
+        {
+            const string saasUserIdType = "sub";
```
Comment (DmitryMorozov228) [inline src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs]: Let's rename saasUserIdType to claimType

Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs]: done

File: src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Softeq.NetKit.Chat.Web.AuthorizationHandling
+{
+    public class UserLoginAuthorizationHandler : IAuthorizationHandler
+    {
+        private readonly IMemberService _memberService;
+
+        public UserLoginAuthorizationHandler(IMemberService componentContext)
```
[the remark this answers is not in the document]
Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs]: done

File: src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs (hunk 1 above)
Comment (eugepae) [reply src/Softeq.NetKit.Chat.Web/AuthorizationHandling/UserLoginAuthorizationHandler.cs]: claimType - not saasUserIdType 😄

**Pull request merged**